pkg/issuer/acme/http/http_test.go

```
@@ -154,7 +156,12 @@ func TestReachabilityCustomDnsServers(t *testing.T) {
 			t.Errorf("failed to write DNS response: %v", err)
 		}
 	})
-	go server.ListenAndServe()
+
+	go func() {
+		if err := server.ListenAndServe(); err != nil {
```

Unrelated change: don't silently fail ListenAndServe, fail test instead.